### PR TITLE
Add a simple Geany portable variant to the FAQ

### PR DESCRIPTION
**Pull request by eht16**
Inspired by https://github.com/geany/geany/issues/3915.

### CODE REVIEW

**Comment by eht16** [comment]
@elextr what do you think?

And it'd be nice if you could check for improvements on my wording :).

File: page_content/documentation/faq.md [hunk 1]
```diff
@@ -201,6 +201,29 @@ indent settings - i.e. you might want one document indented with tabs whilst
 another is open with spaces indentation. See the [manual][8] for details.
 
 
+## How to install a portable version of Geany on Windows (e.g. for USB drives)?
+
+You can use the official installer to install Geany and copy its files to a portable device:
+
+- install Geany from the official installer to the any path on a local disk
+- copy the whole directory to the portable device
+- the uninstall Geany from your local system
```
Comment (elextr) [inline page_content/documentation/faq.md:210]: s/the/then/

File: page_content/documentation/faq.md [hunk 2]
```diff
@@ -201,6 +201,29 @@ indent settings - i.e. you might want one document indented with tabs whilst
...
+  SET XDG_CONFIG_HOME=./config
+  REM Optionally set the language you prefer, a list of available translations can be found on https://www.geany.org/contribute/translation/statistics/
+  SET LANG=en
+
+  start /b bin\geany.exe --config=./config/geany
+  ```
+- if you want to change the GTK theme for another look and feel, see [How to change the GTK theme?][12] below and use the directory `config` for creating the `gtk-3.0` and `themes` directories
+- to install Geany-Plugins proceed like above with the Geany-Plugins installer
+
+There is also a third party package for a portable Geany version, see [Geany Portable for Windows (portableapps.com)][13]. Pleae keep in my mind this package is not maintained by the Geany team and therefore you must make yourself sure to trust it.
```
Comment (elextr) [inline page_content/documentation/faq.md:225]: s/Pleae/Please/

s/to trust/that you can trust/

**Review by elextr — APPROVED** [review]
Sorry missed the call for review, you shoulda pinged.

File: page_content/documentation/faq.md (hunk 1 above)
Comment (eht16) [reply page_content/documentation/faq.md]: Fixed, thanks.

File: page_content/documentation/faq.md (hunk 2 above)
Comment (eht16) [reply page_content/documentation/faq.md]: Fixed, thanks.
Seems like I had my heavy typo day, I even got "faq" wrong in the branch name :).

**Comment by eht16** [comment]
@elextr thanks for the review.

File: page_content/documentation/faq.md (hunk 2 above)
Comment (eht16) [reply page_content/documentation/faq.md]: @elextr I guess I forgot the ping again :).

File: page_content/documentation/faq.md [hunk 3]
```diff
@@ -201,6 +201,29 @@ indent settings - i.e. you might want one document indented with tabs whilst
 another is open with spaces indentation. See the [manual][8] for details.
 
 
+## How to install a portable version of Geany on Windows (e.g. for USB drives)?
+
+You can use the official installer to install Geany and copy its files to a portable device:
+
+- install Geany from the official installer to the any path on a local disk
```
Comment (b4n) [inline page_content/documentation/faq.md:208]:
```suggestion
- install Geany from the official installer to any path on a local disk
```

File: page_content/documentation/faq.md [hunk 4]
```diff
@@ -201,6 +201,29 @@ indent settings - i.e. you might want one document indented with tabs whilst
...
+  SET XDG_CONFIG_HOME=./config
+  REM Optionally set the language you prefer, a list of available translations can be found on https://www.geany.org/contribute/translation/statistics/
+  SET LANG=en
+
+  start /b bin\geany.exe --config=./config/geany
+  ```
+- if you want to change the GTK theme for another look and feel, see [How to change the GTK theme?][12] below and use the directory `config` for creating the `gtk-3.0` and `themes` directories
+- to install Geany-Plugins proceed like above with the Geany-Plugins installer
+
+There is also a third party package for a portable Geany version, see [Geany Portable for Windows (portableapps.com)][13]. Please keep in my mind this package is not maintained by the Geany team and therefore you must make yourself sure that you can trust it.
```
Comment (b4n) [inline page_content/documentation/faq.md:225]:
```suggestion
There is also a third party package for a portable Geany version, see [Geany Portable for Windows (portableapps.com)][13]. Please keep in my mind this package is not maintained by the Geany team and therefore you must make sure yourself that you can trust it.
```
Or any better wording @elextr can give, but the current one sounds a bit weird to me.

Comment (elextr) [reply page_content/documentation/faq.md:225]: Whilst of course @eht16 has universal mind, perhaps we should instead let the user use their own mind ;-)

And then it also becomes their own responsibility for trust.

"Please keep in mind this package is not maintained by the Geany team and it is each user's own decision of its trustworthiness."

PS Excellent wine with dinner, may need rechecking tomorrow ;-)

File: page_content/documentation/faq.md (hunk 3 above)
Comment (eht16) [reply page_content/documentation/faq.md:208]: Obsoleted by @elextr suggestion :).

File: page_content/documentation/faq.md (hunk 4 above)
Comment (eht16) [reply page_content/documentation/faq.md]: Done, thank you both.

**Review by b4n — APPROVED** [review]

File: page_content/documentation/faq.md (hunk 4 above)
Comment (b4n) [reply page_content/documentation/faq.md]: Oh, didn't spot the "my" :)
Sounds better indeed, even sober in the morning (though being woken up at 6 might impede my judgment just as well as wine could have impeded yours …)

Comment (eht16) [reply page_content/documentation/faq.md]: Unfortunately, I didn't have any wine nor did I get up too early, so no more excuses left for me for my probably most "typoful" PR ever :).

**Comment by elextr** [comment]
Assume the checks problems are not related to this PR, it which case it has two approvals so fine.

**Pull request merged**